widgets/src/marshaller/HipieDDL.js

```diff
@@ -60,6 +60,17 @@
                 if (val === undefined) {
                     val = item[rhsKey.toLowerCase()];
                 }
+                //  Symposium AVE Hack 
+                if (val === undefined && rhsKey.indexOf("_AVE") === rhsKey.length - 4 && item.base_count !== undefined) {
+                    var rhsSum = rhsKey.substring(0, rhsKey.length - 4) + "_SUM";
+                    val = item[rhsSum];
+                    if (val === undefined) {
+                        val = item[rhsSum.toLowerCase()];
+                    }
+                    if (val) {
+                        val /= item.base_count;
+                    }
+                }
```

Comment:
will this handle summing multiple avges and dividing it by the sum of multiple base_counts for aggregated wuid-based charts?